City-of-Helsinki/drupal-hdbt-admin
# UHF-8508 Select icon library switch

# [UHF-8508](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8508)


## What was done


* Altered the styles for the color palette selection to match Tom Select.
* Removed mentions of Select2 and Select2 Icon libraries from library overrides and styles.
* Unified naming convention of select list components.

## How to install and test
* Check installation and testing instructions from [platform config PR](https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/530)

## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/530


[UHF-8508]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8508?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
